js/src/components/contentAnalysis/mapResults.js

```diff
@@ -76,6 +79,26 @@ function processResult( mappedResult, mappedResults ) {
 	return mappedResults;
 }
 
+/**
+ * Retrieves the icons and colors for the icon for a certain result.
+ *
+ * @param {string} score The score for which to return the icon and color.
+ *
+ * @returns {object} The icon and color for the score.
+ */
+export function getIconForScore( score ) {
+	switch( score ) {
```

Comment:
The usage of a `switch` statement with immediate returns feels odd. I'd vouch for either setting a variable (that defaults to `{ icon: "seo-score-none", color: colors.$color_grey_disabled };`) in the `switch` and return said variable at the very end. Alternatively, you could replace the switch with `if` statements.